core/src/main/scala/pl/edu/agh/iet/akka_debugging/DistributedStackTraceUtil.scala

```diff
@@ -7,10 +9,10 @@ trait DistributedStackTraceMessage {
   val stackTrace = Thread.currentThread().getStackTrace
 }
 
-trait DistributedStackTrace {
+trait TracedActor {
   self: Actor =>
 
-  var ZMIENNA = -1
+  var MessageWrapperId: UUID = UUID.fromString("00000000-0000-0000-0000-000000000000")
```

Comment:
I don't know - but I think rather no (I think that this advice is run on the same thread as instrumented method)